DOC: attribute PyPy for the idea behind the optimization (#23036)
python/cpython: Doc/whatsnew/3.10.rst

* The ``LOAD_ATTR`` instruction now uses new "per opcode cache" mechanism.
  It is about 36% faster now.  (Contributed by Pablo Galindo and Yury Selivanov
  in :issue:`42093`, based on ideas from Victor Stinner and the original

>   implementation in PyPy.)

Reviewer: I have nothing to do with that! Please remove my name :-)

Reviewer: I think that the only thing that I did was to say to some core devs (INADA-san, Yury and Pablo?) that PyPy has some cool optimizations that we could "easily" reimplement in CPython.

>   in :issue:`42093`, based on ideas from Victor Stinner and the original
>   implementation in PyPy.)

Reviewer 2: ```suggestion
  in :issue:`42093`, based on ideas implemented originally in PyPy and MicroPython.)
```
